apps/examples/wifi_manager_sample/wm_test.c

```diff
@@ -645,22 +645,21 @@ wm_parse_commands(struct options *opt, int argc, char *argv[])
 		opt->func = wm_scan;
 	} else if (strcmp(argv[2], "auto") == 0) {
```

Comment:
why don't you use strncmp with length of 5?

Reply:
I think that using strcmp is ok because argv always contains null character .